src/main/java/aquality/selenium/browser/RemoteBrowserFactory.java

```diff
@@ -60,14 +60,13 @@ protected RemoteWebDriver getDriver() {
         private final Factory defaultClientFactory = Factory.createDefault();
         private final Duration timeoutCommand = timeoutConfiguration.getCommand();
 
-        @Override
-        public Builder builder() {
-            return defaultClientFactory.builder().readTimeout(timeoutCommand);
+        public HttpClient createClient(URL url) {
+            return defaultClientFactory.createClient(ClientConfig.defaultConfig().baseUrl(url).readTimeout(timeoutCommand));
         }
 
         @Override
-        public HttpClient createClient(URL url) {
-            return this.builder().createClient(url);
+        public HttpClient createClient(ClientConfig clientConfig) {
+            return defaultClientFactory.createClient(clientConfig);
```

Comment:
could you specify the timeout here please?
`clientConfig.readTimeout(timeoutCommand)`

Reply:
Fixed